.gitignore

```diff
@@ -1,3 +1,5 @@
+vendor
```

Comment:
IMHO we shouldn't remove the vendor dir.
If a dependency or its host goes down, we can't build Gitea anymore.